language/resource-api/README.md

```diff
@@ -495,19 +507,21 @@ The `stdin_source:` keyword argument takes the following values:
 * `:io`: the input of the process is connected to the IO object specified in `stdin_io:`.
 * `:none`: the input of the process is closed, and the process will receive an error when trying to read input. This is the default.
 
-#### Character encoding
+##### Character encoding
+
+To support the widest array of platforms and use cases, character encoding of a provider's inputs and outputs need to be considered. By default the commands API follows the Ruby model of having all strings tagged with their current [`Encoding`](https://ruby-doc.org/core/Encoding.html), and uses the system's current default character set for I/O. This means that strings read from commands might be tagged with non-UTF-8 character sets on input and UTF-8 strings transcoded on output.
 
-To support the widest array of platforms and use-cases, character encoding of a provider's inputs, and outputs needs to be considered. By default, the Commands API follows the ruby model of having all strings tagged with their current [`Encoding`](https://ruby-doc.org/core/Encoding.html), and using the system's current default character set for I/O. That means that strings read from Commands might be tagged with non-UTF-8 character sets on input, and UTF-8 strings might be transcoded on output.
+[TODO: is it IO or I/O? Or are they different?]
```

Comment:
In `stdin_source:`, and the `_destination:` keywords, 'IO' specifically refers to a class (http://ruby-doc.org/core-2.3.1/IO.html) . Here 'I/O' refers to the process input and output streams.